deployments/embed.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2021 Intel Corporation. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package embed generates plugin DaemonSet object out of plugin yaml.
+package embed
```

Comment:
This package gets imported as `deployments`. Is it possible to name the package the same way?

Reply:
done

Reply:
hm, this triggered this CI error:
```
go mod download
go: finding module for package github.com/intel/intel-device-plugins-for-kubernetes/deployments
make: *** [Makefile:46: go-mod-tidy] Error 1
github.com/intel/intel-device-plugins-for-kubernetes/pkg/controllers/dsa imports
	github.com/intel/intel-device-plugins-for-kubernetes/deployments: no matching versions for query "latest"
Error: Process completed with exit code 2.
```

running ```make go-mod-tidy``` didn't show this error for some reason.

Reply:
should be fixed now